tsconfig.json

```diff
@@ -0,0 +1,29 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "rootDir": "./",
+    "outDir": "dist/tsc/",
+    "noEmitOnError": true,
+    "sourceMap": true,
+    "module": "amd",
```

Comment:
This is wrong. Since the project is `dojo` based `amd` is required for loading the modules